README.md

```diff
@@ -14,6 +14,18 @@ Also, a friendly reminder that contributions are not limited to code - it is abo
 
 **Prerequisites**: Go version 1.7 or greater.
 
+### For Go >=1.11:
```

Comment:
I don't think we need separate instructions for go > 1.11. `go get` would work for it as well.

Reply:
Done!